content/en/docs/Reference/_index.md

```diff
@@ -6,4 +6,5 @@ description: >
   Reference guides for OpenCue tools and interfaces for all users
 ---
 
+*   [cueadmin](/docs/reference/commands/cueadmin)
```

Comment:
Should it be capitalized to match the item below, or doesn't matter?

Reply:
Thanks for checking, it does matter. I'm using lowercase to refer to the tool as that's generally the way command-line tools are referred to. It should help as a subtle indicator to the reader that this is where you go to find info on a command-line tool when you don't know what tools are available. When we put the initial Wiki together, I probably wasn't entirely clear on Cueadmin's capabilities.

My understanding is that Cueadmin is only a command-line tool, so we should consistently refer to it as 'cueadmin' going forward.

Reply:
👍